### PR TITLE
open-cluster-management/backlog#1580 start internal refresh/publish doc.

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: _pages/arch_builds.md [hunk 1]
```diff
@@ -1,12 +1,103 @@
...
+   
+5. If the branch is not the one that you want to build (for example, the line is `0 0 * * 0 /root/sync/acm_sync_asciidoc.sh 2.1 2.1_stage > /root/sync/2.1_acm_cron_log`), change it to the one that you want. 
+  1. Move your cursor to the beginning of `stage` in the branch name of the entry (2.1_stage). 
+  2. Use the `x` key to delete the characters in `stage`.
+  3. Type the `i` key to go into insert mode.
+  4. Insert `prod` where you just removed `stage`.
+  5. Press the `esc` key to leave insert mode. 
+
+6. Enter `:wq` to save the `crontab file and close it.
+
```
Comment (swopebe) [inline _pages/arch_builds.md:76]: Did you mean to put this part in code block? I think extra character here.

File: _pages/arch_builds.md [hunk 2]
```diff
@@ -1,12 +1,103 @@
...
+   redhat1!
+   ```
+
+3. Run the following to change your directory: 
+
+   ```
+   cd /root/sync/
+   ```
+
+4. Run the following command, where `X.X doc_branch` is the version number and either `stage` for daily work, or `prod` for a live refresh or stage build. **Note:** The branch setting here must match the branch setting that is set in the `crontab` file. See the "Confirm build branch" section for information about how to verify or change this. For example, if the branch in the `crontab` file for your release is set to `2.1_prod`, then your branch in this command must be `2.1_prod`.
```
Comment (swopebe) [inline _pages/arch_builds.md]: Can we be a bit more concise on the paragraphs here?

File: _pages/arch_builds.md (hunk 1 above)
Comment (cadawson) [reply _pages/arch_builds.md:76]: Fixed

**Review by cadawson — APPROVED** [review]

File: _pages/arch_builds.md [hunk 3]
```diff
@@ -1,12 +1,122 @@
...
+
+7. Confirm that the commit number that you noted in step 5 is listed beside the stage line for each book (in this example, `5bfee54`). If it does not, select the down arrow beside the stage version of the book select **Rebuild** for each book that requires it. 
+
+   You can review build logs to check for failures.
+
+8. **Important** Only if you are publishing live. After all of the builds for the books are clean and the commits match your build, you are ready to publish. See the following steps to _Publish a GA version_.
+
+## Publish a GA version
+   
+1. When you have completed all of the previous steps for the `prod` branch of your release, you can publish by selecting **Publish** beside each book on the Pantheon UI. We do not publish the `*.stage` branches live. **Note:** If the publish button is grayed out, make sure that you are logged in to Pantheon.
```
Comment (dockerymick) [inline _pages/arch_builds.md]: Remove `beside each book`. I suggest updating the sentence to this: 

When you have completed all of the previous steps for the `prod` branch of your release, you can publish by selecting **Publish** for each book on Pantheon.

**Review by dockerymick — APPROVED** [review]

**Pull request merged**